### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Deploy Web App_Color Contrast: In Dark Theme, Color Contrast for ‘App Service pricing details’ link is less than 4.5:1

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3343]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till Create button and press enter
11) Open color contrast analyser and verify the color contrast for ‘App Service pricing details’ link

**Actual Result:** 
In dark Theme, Color Contrast for ‘App Service pricing details’ link is 3.9:1 which is less than 4.5:1

**Expected Result:** 
In dark Theme, Color Contrast for ‘App Service pricing details’ link should be greater than or equal to 4.5:1

**User Impact:** 
Users with low vision will not be able to see the text properly if the color contrast ratio is less than required

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_19_Color Contrast for ‘App Service pricing details’ link is less than 4 51](https://user-images.githubusercontent.com/41109135/62205987-5ef05180-b3ae-11e9-95a6-7352026b81e0.png)

**Comment by Flanker32** [comment]
@Tanushagottipati The front and background color of components like link is controlled by Eclipse theme, which is out of the scope of our toolkits,,so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a screenshot of Eclipse UI, and we can see the color contrast for link is lower than 4.5

![image](https://user-images.githubusercontent.com/12445236/64943756-2d204500-d89f-11e9-8f38-fba005ba5b02.png)



Besides, eclipse provide multi themes and the ratio is more than 4.5 in other theme like Light, here is the screen shot.

![image](https://user-images.githubusercontent.com/12445236/64943625-d61a7000-d89e-11e9-8535-156f00e57180.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium, users with low vision will not be able to see the text properly if the color contrast ratio is low.
- Quality of experience: None, Eclipse provides multi themes and the contrast ratio will be larger than 4.5 in other themes like Light
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Eclipse which has been linked to AAT, please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551118

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by Manisha370** [comment]
Similar issue is observed in "New HDInsight Scala Project" page
In dark theme, the color contrast ratio for links are less than 4.5:1

![In dark theme links has poor contrast ratio](https://user-images.githubusercontent.com/51435937/79111029-5f444780-7d99-11ea-89d5-b04e0771c52d.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749661
